test/unit/lodash.spec.js

```diff
@@ -0,0 +1,612 @@
+'use strict';
+
+var chai = require('chai');
+var expect = chai.expect;
+var _ = require('lodash');
+var datasets = require('../../app/datasets');
+
+describe('Lodash training', function ()
+{
+    before(function ()
+    {
+        Array.prototype.checkIfArrayIsUnique = function ()
```

Comment:
If we wan't to use prototype - we need to quiet `jshint` (because we find it in the report errors  - RealSkill)

```      
/*jshint -W121*/
Array.prototype.checkIfArrayIsUnique = function ()
```